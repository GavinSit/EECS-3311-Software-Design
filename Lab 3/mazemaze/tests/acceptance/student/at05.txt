runrunrun
pewpewpew
new_game(sdf)
new_game(medium)
move(oninlife)