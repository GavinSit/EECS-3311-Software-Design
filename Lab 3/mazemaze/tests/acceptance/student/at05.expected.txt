Syntax Error: specification of command executions cannot be parsed
Unknown identifier oninlife (Line 5, Column 6)
